### PR TITLE
Node4 support

### PR DESCRIPTION
**Pull request by nickbalestra**

### CODE REVIEW

**Comment by nickbalestra** [comment]
- [ ] compile.test need to be rewritten, everything else should be now node4 ready

**Comment by nickbalestra** [comment]
@matteofigus all 💚

File: packages/oc-template-handlebars-compiler/test/compile.test.js [hunk 1]
```diff
@@ -1,89 +1,38 @@
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
```
Comment (matteofigus) [inline packages/oc-template-handlebars-compiler/test/compile.test.js]: :rofl:

**Comment by nickbalestra** [comment]
Yes, I've created the issue #24 to address those little cleanups/improvements

**Pull request merged**